manager/src/main/java/com/redhat/developer/manager/requests/ConnectorRequest.java

```diff
@@ -0,0 +1,19 @@
+package com.redhat.developer.manager.requests;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.redhat.developer.manager.models.Connector;
+
+public class ConnectorRequest {
```

Comment:
At some point we'll probably want to break this out into an `api` package e.g `eventthing-api` so that users of the service can easily import just the public facing API into their Java apps.

So we might want to be a little careful about having the internal domain model in the public facing API. It's not a huge probably right now, but it might be easier to start with a mapper that converts the API objects into internal domain objects and vice-versa.